docs/how-to-guides/environment-setup-and-maintenance/setup-isis-data.md

```diff
@@ -0,0 +1,153 @@
+# Setting up ISIS data for a new mission
+
+Every spacecraft camera that ISIS supports requires a directory in the [`ISIS data area`](../../how-to-guides/environment-setup-and-maintenance/isis-data-area.md). It stores the camera position, orientation, sensor properties, etc.; data that is later added to an image to process with [spiceinit](https://isis.astrogeology.usgs.gov/Application/presentation/Tabbed/spiceinit/spiceinit.html). This page documents how to set up such a directory for a new mission, with the Chandrayaan-2 Lunar orbiter data serving as an example.
+
+The environmental variable ISISDATA points to the top-most directory in the ISIS data area. We will create there the subdirectory `${ISISDATA}/chandrayaan2`, and inside of it there will be a directory named ``kernels`` that will have the above-mentioned metadata, which in the planetary data community is called [SPICE kernels](https://naif.jpl.nasa.gov/naif/index.html).
+
+The ``kernels`` directory has subdirectories with names such as ``spk``, ``ck``, ``ik``, etc., whose meaning will be discussed shortly. Each of these must have one more index files, in plain text, with a name such as ``kernels.0000.db``, that enumerates the SPICE kernels and some of their properties. This file name normally has four digits, and the number gets incremented if there is more than one index.
+
+For ``spk`` and ``ck`` kernels, ISIS3 provides a dedicated tool called [kerneldbgen](https://isis.astrogeology.usgs.gov/Application/presentation/Tabbed/kerneldbgen/kerneldbgen.html). For the others, this index file needs to be set up manually.
+
+## SPK kernels
+
+SPK stands for Spacecraft Position Kernels. They contain the ephemeris (position and velocity) information for spacecraft, planetary bodies, etc. The kernels usually have the ``.bsp`` extension. Create the directory:
+
+```sh
+
+  $ISISDATA/chandrayaan2/kernels/spk
+
+```
+
+and copy there these files. Run a command as:
+
+```sh
+
+   cd ${ISISDATA}/chandrayaan2/kernels/spk
+
+   kerneldbgen                                       \
+    to = '$chandrayaan2/kernels/spk/kernels.????.db' \
+    type = SPK                                       \
+    recondir = '$chandrayaan2/kernels/spk'           \
+    reconfilter = 'ch2*.bsp'                         \
```

Comment:
determining the quality might be tricky, usually kernels ship with a text file (ckinfo.txt, spkinfo.txt) that describe this when trying to make a new kerneldb. Might be worth mentioning here. LRO [has an example ](https://naif.jpl.nasa.gov/pub/naif/pds/data/lro-l-spice-6-v1.0/lrosp_1000/data/spk/spkinfo.txt), but CHA-2 is weird because they are not managed by naif.